lib/js/components/Pill/Pill.vue

```diff
@@ -0,0 +1,196 @@
+<template>
+	<div
+		class="pill"
+		:class="{ '-x-small': size === PILL_SIZES.X_SMALL, [colorClassName]: true }"
+		:title="label"
+	>
+		<icon
+			v-if="leftIcon"
+			class="pill__leftIcon"
+			:icon="leftIcon"
+			:size="size === PILL_SIZES.X_SMALL ? ICON_SIZES.XXX_SMALL : ICON_SIZES.XX_SMALL"
+		/>
+		<span class="pill__label">{{ label }}</span>
+		<icon-button
```

Comment:
Is it alway the same size? Also there is no deletable small pill version in figma. Should we allow for that?

Reply:
Good catch, asked UX: https://www.figma.com/file/HFY6waq7kWsr3rj8a0ghXZ?node-id=1499:118929#217941952

Reply:
UX confirmed that we should hide it on `x-small`: [`831f5b0` (#155)](https://github.com/bethinkpl/design-system/pull/155/commits/831f5b0c257749759f541fab5e944ffebc6016d5)